modules/virt-removing-interface-from-nodes.adoc

```diff
@@ -30,19 +32,29 @@ spec:
     node-role.kubernetes.io/worker: "" <3>
   desiredState:
     interfaces:
-      - name: br1
-        type: linux-bridge
-        state: absent <4>
+    - name: br1
+      type: linux-bridge
+      state: absent <4>
+    - name: eth1
+      type: ethernet <5>
+      state: up <6>
+      ipv4:
+        dhcp: true <7>
+        enabled: true <8>
 ----
 <1> Name of the policy.
 <2> Optional: If you do not include the `nodeSelector` parameter, the policy applies to all nodes in the cluster.
 <3> This example uses the `node-role.kubernetes.io/worker: ""` node selector to select all worker nodes in the cluster.
 <4> Changing the state to `absent` removes the interface.
+<5> The type of interface. This example creates an Ethernet networking interface.
+<6> The requested state for the interface after creation.
+<7> Optional: If you do not use `dhcp`, you can either set a static IP or leave the interface without an IP address.
+<8> Enables `ipv4` in this example.
 
 . Update the policy on the node and remove the interface:
 +
 [source,terminal]
 ----
 $ oc apply -f <br1-eth1-policy.yaml> <1>
 ----
-<1> File name of the policy manifest.
+1> File name of the policy manifest.
```

Comment:
Looks like the preceding &lt sign was removed by mistake, so in the build preview it appears as "1>" rather than "1" framed inside a circle.